src/servers/main.cc

```diff
@@ -807,6 +812,7 @@ Parse(TRITONSERVER_ServerOptions** server_options, int argc, char** argv)
   int32_t exit_timeout_secs = 30;
   int32_t repository_poll_secs = repository_poll_secs_;
   int64_t pinned_memory_pool_byte_size = 1 << 28;
+  int32_t buffer_manager_thread_count = 0;
```

Comment:
Default is 1. Don't confuse the argument value with what you use to initialize the async work queue.